lib/authorizations.js

```diff
@@ -0,0 +1,35 @@
+/*!
+ * Copyright (c) 2019-2023 Digital Bazaar, Inc. All rights reserved.
+ */
+import * as bedrock from '@bedrock/core';
+import forwarded from 'forwarded';
+import {verify} from '@bedrock/turnstile';
+
+const {config, util: {BedrockError}} = bedrock;
+
+export async function authorizeRegistration({req}) {
+  const cfg = config['account-http'];
+  const {authorizationRequired} = cfg.registration;
+  const {authorization} = req.body;
+  if(!(authorization?.type === authorizationRequired)) {
```

Comment:
IMO, this is clearer:

```suggestion
  if(authorization?.type !== authorizationRequired) {
```